product-updates/security-manager-2-release-note.md

```diff
@@ -8,13 +8,31 @@ SysKit Security Manager is centralized __SharePoint and Office 365 security repo
 [Try SysKit Security Manager Free](https://www.syskit.com/products/security-manager/download/)
 
 __Product version:__ 2.0.0  
-__Build number:__ 590      
+__Build number:__ 599     
 __Release date:__ May 16, 2018  
 
 ## Features
-* __SharePoint Online and On-Premises Snapshots__ are now available in SysKit Security Manager! In addition to being able to manage SharePoint Online and On-Premises permissions, you can now take snapshots of all your site collections and Office 365 groups.
+* __SharePoint Online and On-Premises Snapshots__ are now available in SysKit Security Manager! In addition to being able to manage SharePoint Online and On-Premises permissions, you can now take snapshots of all your site collections and Office 365 groups. This is required in order to use more advanced __Permission Reports__
+
+* Use the __Permissions Reports__ screen to gather valuable information about structure, granted permissions, broken permissions inheritance, and more. We broke them down into four categories. For detailed information on every report see [this article](#internal/get-to-know-security-manager/permissions-reports-screen).
+
+    * __Permissions reports:__ 
+    
+      Auditing SharePoint permissions can be a complex and time-consuming task. Syskit Security Manager allows administrators to drill down and view all permissions, explore permissions history, and create site and user specific reports.
+      
+    * __User reports:__
+    
+       These reports can either give you a __summary__ or a __detailed view__ on the permissions that principal has within a particular site collection. 
+
+    * __Externally Shared:__
+
+      A bundle of reports concerning externally shared content and external users.  
+      They will give you information about the content that is shared with external users where you can see what permissions those users have on the content, a list of content shared via anonymous links, and even a list of all unlicensed (external) users on your tenant. 
+
+    * __Cleanup:__
+
+      It is a known thing that SharePoint gets cluttered. Users get disabled, they lose permissions, SharePoint groups lose their members and so on. In all these cases it is recommended to manage (__cleanup__) such groups and users. Explore [our Cleanup reports](#internal/get-to-know-security-manager/permissions-reports-screen#cleanup) and find out what you can do. 
```

Comment:
I think something like _... In cases like this ..._ is a more natural construction.
_recommended to manage_ seems a bit odd of a phrase choice, since since it is very much required to keep an overview of all the states you noted, and you should be able to manage them easily. I just don't have an idea what to substitute it with, so that it is still succinct.

_what you can do about those groups and users._, [just a weak suggestion]